robot/Cumulus/resources/locators_48.py

```diff
@@ -6,6 +6,7 @@
     'breadcrumb-link':"//a[@title='{}' and contains(@class,'BreadCrumbItem')]",
     'obj-header':"//h1//*[text()='{}']",
     'button-title':"//button[@title='{}']",
+    'button-with-text':'//button[contains(text(),"{}")]',
```

Comment:
curious, we already have a locator for //button[text()='{}']locator, can't we just use the same

Reply:
This requires contains because the text is not exact and has spaces in between. Trim did not work either.